x/mongo/driver/topology/topology_options_test.go

```diff
@@ -9,73 +9,49 @@ package topology
 import (
 	"fmt"
 	"testing"
-	"time"
 
 	"github.com/stretchr/testify/assert"
-	"go.mongodb.org/mongo-driver/x/mongo/driver/connstring"
+	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
-func TestOptionsSetting(t *testing.T) {
```

Comment:
I don't think this test is needed anymore since we no longer support functional config options.